pkg/config/config.go

```diff
@@ -39,6 +39,10 @@ type Config struct {
 		QueueNum string `default:"100"`
 	}
 
+	Redis struct {
```

Comment:
Change redis to queue will be better.

The queue addr can be redis or others.

Queue struct {
    Type string `default:"redis"`
    Addr string `default:"redis://localhost:6379"`
}